<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCompany">
    <property name="title">
     <string>公司</string>
    </property>
    <addaction name="actionRegisterCompany"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="actionLoginUser"/>
    <addaction name="actionLogoutUser"/>
    <addaction name="actionHomePage"/>
   </widget>
   <addaction name="menuCompany"/>
   <addaction name="menuUser"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRegisterCompany">
   <property name="text">
    <string>注册公司</string>
   </property>
  </action>
  <action name="actionLoginUser">
   <property name="text">
    <string>登录</string>
   </property>
  </action>
  <action name="actionLogoutUser">
   <property name="text">
    <string>退出登录</string>
   </property>
  </action>
  <action name="actionHomePage">
   <property name="text">
    <string>主页</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
